src/views/Game.js

```diff
@@ -1,11 +1,11 @@
 /* eslint-disable react/prop-types */
 import React, { useContext, useEffect } from 'react';
 import { GameContext } from 'contexts/GameContext';
+import { ToastContainer, toast } from 'react-toastify';
```

Comment:
Non-blocker: FYI mantine it's own built in [notification system](https://v1.mantine.dev/core/notification/)

Reply:
Got it, we can switch over to that then